extensions/amp-analytics/0.1/test/test-visibility-impl.js

```diff
@@ -638,6 +638,66 @@ describe('amp-analytics.visibility', () => {
       });
     });
 
+    it('"hidden" trigger should work with duration condition', () => {
```

Comment:
In fact, I think the V1 code for hidden trigger does not take non-duration condition into consideration either.